src/tests/ftest/util/dmg_utils_base.py

```diff
@@ -533,6 +535,15 @@ def __init__(self):
                 self.ranks = FormattedParameter("--ranks={}")
                 self.verbose = FormattedParameter("--verbose", False)
 
+        class CleanupSubCommand(CommandWithParameters):
+            """Defines an object for the dmg system cleanup command."""
+
+            def __init__(self):
+                """Create a dmg system cleanup command object."""
+                super().__init__("/run/dmg/system/cleanup/*", "cleanup")
+                self.machinename = FormattedParameter("{}", None)
```

Comment:
Nothing particularly wrong here but if this parameter is not preceded by any text then the inherited `BasicParameter` class could be used.  If the parameter is positional then the `PositionalParameter` class could be used to ensure a specific argument order.